lib/shared/require-or-import.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+try {
+  module.exports = require('./require-or-import-es6');
```

Comment:
Node.js <10 errors out with a SyntaxError when loading a script that uses `import()`. By adding another layer and falling back to a `require()`-only implementation (below), we can keep supporting all versions all the way back to Node.js 0.10.